tests/basic/longevity_test.go

```diff
@@ -248,15 +262,15 @@ func emailEventTrigger(wg *sync.WaitGroup,
 		if isTriggerEnabled && time.Since(lastInvocationTime) > time.Duration(waitTime) {
 			// If trigger is not disabled and its right time to trigger,
 
-			logrus.Infof("Waiting for lock for trigger [%s]\n", triggerType)
+			dash.Infof("Waiting for lock for trigger [%s]\n", triggerType)
 			emailTriggerLock.Lock()
-			logrus.Infof("Successfully taken lock for trigger [%s]\n", triggerType)
+			dash.Infof("Successfully taken lock for trigger [%s]\n", triggerType)
 
 			triggerFunc()
-			logrus.Infof("Trigger Function completed for [%s]\n", triggerType)
+			dash.Infof("Trigger Function completed for [%s]\n", triggerType)
```

Comment:
same here, you can remove this and above from dashboard and just log to file

Reply:
fixed